tests/testthat/test-ui-sunburst.R

```diff
@@ -3,10 +3,14 @@ context("canvasXpress Web Charts - Sunburst")
 
 test_that("cXsunburst1", {
     check_ui_test(cXsunburst1())
+
+    warning("Clicking on legend misplace the labels on chart")
```

Comment:
change to fail - and to message 'Legend click results in label placement issues'